src/app/personside/infotabs/oversikt/OppfolgingOversikt.tsx

```diff
@@ -48,7 +52,8 @@ function OppfolgingVisning({ oppfolging }: { oppfolging: Oppfolging }) {
             <Normaltekst>
                 <Bold>Veileder:</Bold>
             </Normaltekst>
-            <Normaltekst>{veilederNavn}</Normaltekst>
+            {veilederNavn}
+            {veilederIdent}
```

Comment:
Nå vil det vel havne på to linjer..?

Reply:
Hva med å bruke den gamle løsningen din men bare omgi de som skal være lette å kopiere med `<span>`?

`<span>${oppfolging.veileder.navn}</span> <span>(${oppfolging.veileder.ident})</span>`

Reply:
Kan teste

Reply:
Nei, ikke ønsket effekt. De ønsker å kunne dobbeltklikke på navnet for å merke og kopiere, men dersom det ligger på en linje så vil en dobeltklikk ta med ident også, derfor to linjer. usikker på om det kan løses på annen måte

Reply:
aha, jammen da så!